repos/c2sm/packages/py-gt4py/package.py

```diff
@@ -15,40 +15,46 @@ class PyGt4py(PythonPackage):
 
     url = "ssh://git@github.com/GridTools/gt4py.git"
 
-    version('functional', branch='functional', git=url)
+    version('main', branch='main', git=url)
+    version('1.0.1', tag='v1.0.1', git=url)
```

Comment:
So the TL;DR of this is, our icon4py and gt4py developments are currently in flux, and we won't have usable tags for a week or two. I'm just specifying the main branches for both here so that we have at least something working for now. This will not be the version that goes out to the users.